install_config/revhistory_install_config.adoc

```diff
@@ -4,6 +4,31 @@
 :data-uri:
 :icons:
 :experimental:
+== Wed Apr 27 2016
+
+// tag::install_config_wed_apr_27_2016[]
+[cols="1,3",options="header"]
+|===
+
+|Affected Topic |Description of Change
+//Wed Apr 27 2016
+n|link:../install_config/persistent_storage/persistent_storage_nfs.html[Configuring Persistent Storage -> Persistent Storage Using NFS]
```

Comment:
Stray `n` at the front of this line? It doesn't appear to do anything in the rendered version, but... weird.